### PR TITLE
📦 NEW: NonVerbose

### PR DESCRIPTION
**Pull request by vzsky**
Solve issue #28 
add options :
   -b for borderless table -> look a lot cleaner
   -q for quiet -> display no unrelated data, only the table

### CODE REVIEW

**Comment by ahmadawais** [comment]
Howdy, can you address the conflicts and also post a screenshot :)

File: index.js [hunk 1]
```diff
@@ -24,14 +24,17 @@ const {
 	colored,
 	singleStates,
 	coloredStates,
-	style
+	style,
+	tab_borderless
```
Comment (ahmadawais) [inline index.js]:
```suggestion
minimal
```

File: utils/cli.js [hunk 2]
```diff
@@ -41,6 +41,16 @@ module.exports = meow(
 				type: "string",
 				default: "cases",
 				alias: "s"
+			},
```
Comment (ahmadawais) [inline utils/cli.js:46]: Wouldn't it be better just to have one `minimal` options that does all of this?

**Comment by vzsky** [comment]
Screen Shot 2563-03-25 at 10 16 09
This is the screenshot of minimal mode

File: utils/cli.js [hunk 3]
```diff
@@ -41,6 +41,11 @@ module.exports = meow(
...
+			minimal: {
+				type: "boolean",
+				defualt: true,
```
Comment (ahmadawais) [inline utils/cli.js]: Default true is not the right thing here.

File: utils/init.js [hunk 4]
```diff
@@ -2,7 +2,8 @@ const welcome = require("cli-welcome");
 const pkgJSON = require("./../package.json");
 const updateNotifier = require("update-notifier");
 
-module.exports = async () => {
+module.exports = async (quiet) => {
+	if (quiet) return
```
Comment (ahmadawais) [inline utils/init.js:6]: Why change it to `quiet` now?

**Comment by ahmadawais** [comment]
🔗 https://github.com/ahmadawais/corona-cli/releases/tag/3.7.0
🔗 https://www.npmjs.com/package/corona-cli

**Pull request merged**